test/feature-benchmark/mzcompose.py

```diff
@@ -423,10 +423,14 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             sys.exit(1)
 
 
-def _are_regressions_justified(this_tag: str, baseline_tag: str) -> tuple[bool, str]:
-    if not _tag_references_release_version(
-        this_tag
-    ) or not _tag_references_release_version(baseline_tag):
+def _are_regressions_justified(
+    this_tag: str | None, baseline_tag: str
+) -> tuple[bool, str]:
+    if (
+        not this_tag
```

Comment:
I would rather prefer this check in `_tag_references_release_version`. In there, it will also check `baseline_tag` not being `None`. Would you mind changing that?